### PR TITLE
Add support for Symfony 7

### PR DESCRIPTION
**Pull request by relthyg**

### CODE REVIEW

**Comment by eymengunay** [comment]
Hi @relthyg
Thanks a lot for your pull request!
Although I see that you drafted it. Was that intentional by any chance?

**Comment by relthyg** [comment]
Hi @eymengunay, thanks for reaching out! Yes, that was intentional, as I'm not using the PR version in production yet. I wanted to release it for merging as soon as I see there are no issues.

**Comment by relthyg** [comment]
Hi @eymengunay, the PR is now up for grabs!

**Comment by eymengunay** [comment]
Hey @relthyg 
That is awesome! Thank you for your PR, I highly appreciate it!

**Comment by relthyg** [comment]
Thanks for merging! Would you mind tagging a new release?

**Comment by relthyg** [comment]
Always a pleasure!

File: Form/Type/HoneypotType.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function __construct(RequestStack $requestStack, HoneypotManager $honeypo
     /**
      * {@inheritdoc}
      */
-    public function buildForm(FormBuilderInterface $builder, array $options)
+    public function buildForm(FormBuilderInterface $builder, array $options): void
```
Comment (acrobat) [inline Form/Type/HoneypotType.php:61]: @eymengunay adding return types in non-final classes is actually a bc break if it's done in a minor. This currently break applications extending this class after the update to 2.2.0

**Pull request merged**